modules/ROOT/pages/managing-business-groups.adoc

```diff
@@ -134,10 +270,11 @@ The business group ID is `XXXXXXX-XXXX-XXXX-XXXX-XXXXXXX` in the example, and it
 +
```

Comment:
Escape the URL on line 267.

Reply:
@hannanelson This is intentionally not escaped. It actually works as intended. If you click that link it shows you your business group ID.